lib/tessel/deployment/rust.js

```diff
@@ -28,7 +28,7 @@ var exportables = {
       var cargoToml = toml.parse(fs.readFileSync(path.join(pushdir, 'Cargo.toml'), 'utf8'));
 
       if (cargoToml.package) {
-        basename = path.basename(program);
+        basename = pushdir;
```

Comment:
:+1: I think I remember that property not being defined but didn't fix it because it wasn't breaking anything and then just forgot. 🙈